.gitignore

```diff
@@ -25,3 +25,37 @@ docs/_site/
 # VSCode files
 .vscode/
 
+bin/test/seedu/address/logic/LogicManagerTest.class
```

Comment:
All these code should be shortened to *.class. Remember to add a section header to indicate you ignoring all class files